README.md

```diff
@@ -1,43 +1,58 @@
+Now with the proof generated by ZooKyc, Gerry goes back to ZooLender (in `gpc-verifier-client`). ZooLender verifies the proof, and lends Gerry the money.
+
+## Example code
+
+- How to issue PODs: [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/pod-issuer-server/src/routes/debug.ts#L92-L114)
+- Proof request (what you want to prove, including proof configuration, and optionally membership lists, external nullifier and watermark): [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/gpc-verifier/app/hooks/useProofRequest.ts)
+- How to create GPC proofs (based on the proof request): [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/gpc-prover-client/app/util/generateProof.ts)
+- How to verify GPC proofs: [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/gpc-verifier/app/util/verifyProof.ts)
 
 ## What's inside?
 
-This Turborepo includes the following packages/apps:
+This Turborepo includes the following apps:
 
-### Apps and Packages
+### Apps
 
-- `docs`: a [Next.js](https://nextjs.org/) app
-- `web`: another [Next.js](https://nextjs.org/) app
-- `@repo/ui`: a stub React component library shared by both `web` and `docs` applications
-- `@repo/eslint-config`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
-- `@repo/typescript-config`: `tsconfig.json`s used throughout the monorepo
+- `pod-issuer-client`: http://localhost:3000, deployed at https://zukyc-issuer-client.vercel.app. It includes:
+  - [ZooGov](https://zukyc-issuer-client.vercel.app/gov): Govenment ID POD issuer.
+  - [ZooDeel](https://zukyc-issuer-client.vercel.app/deel): Paystub POD issuer.
+  - [ZooAdmin](https://zukyc-issuer-client.vercel.app/debug): For debugging purposes, it allows you to generate ID PODs and Paystub PODs with the information you provide.
+- `pod-issuer-server`: http://localhost:8080, deployed at https://zukyc-issuer-server.vercel.app.
+  - This is the Express server for pod-issuer-client. It provides the APIs to issues PODs to the user.
+- `gpc-prover-client` (ZooKyc): http://localhost:3002, deployed at https://zukyc-gpc-prover-client.vercel.app.
+  - This is a Next.js app which uses the GPC library to generate ZK proofs. GPCs can prove properties of one POD or serveral PODs together. PODs can be proven to be owned by the prover, using their Semaphore identity.
+- `gpc-verifier` (ZooLender): http://localhost:3001, deployed at https://zukyc-gpc-verifier.vercel.app.
+  - This is a Next.js app which uses the GPC library to verify ZK proofs.
+  - I didn't build a server side app for this, because the verification code would be similar. But in reality, the proof needs to be verified on the server side too.
```

Comment:
This sudden shift to "I" feels out of place.  I think the content of this bullet is good, I just suggest rephrasing this in terms of what this demo doesn't do, rather than what "I" didn't do.